src/github.com/stellar/go-horizon/init_versions.go

```diff
@@ -0,0 +1,48 @@
+package horizon
+
+import (
+  "fmt"
+  "io/ioutil"
+  "net/http"
+  "os/exec"
+  "strings"
+  "encoding/json"
+)
+
+func initStellarCoreVersion(app *App) {
+  if app.config.StellarCoreUrl == "" {
+    return
+  }
+
+  resp, err := http.Get(fmt.Sprint(app.config.StellarCoreUrl,"/info"))
+
+  if err != nil {
+    app.log.Panic(app.ctx, err)
+  }
+
+  defer resp.Body.Close()
+  contents, err := ioutil.ReadAll(resp.Body)
+  if err != nil {
+    app.log.Panic(app.ctx, err)
+  }
+
+  var responseJson map[string]*json.RawMessage
+  err = json.Unmarshal(contents, &responseJson)
+
+  var serverInfo map[string]string
+  err = json.Unmarshal(*responseJson["info"], &serverInfo)
+  app.coreVersion = serverInfo["build"]
+}
+
+func initHorizonVersion(app *App) {
```

Comment:
This might work at present, but it will fail when the binary is not executed from a child directory of the repo.

I think a better route would be to have a manually defined version, with an optional, overridable variable for the git revision as described at http://stackoverflow.com/questions/11354518/golang-application-auto-build-versioning.

We can then have travis set the appropriate linker flags to override the var.  Does that make sense?